Common/Distribution/LK8000/_System/CREDITS.TXT

```diff
@@ -3,6 +3,7 @@
 [Latest software changes]
 In summary what's up in current branch:
 
+Airspace Class Glider Sector "GldSec" added (GSEC open airspace, GLIDING OpenAIP)
```

Comment:
You also need to change the first line of the file, otherwise it wont be triggered on first run..